prefect_databricks/flows.py

```diff
@@ -314,7 +315,7 @@ def jobs_runs_submit_and_wait_for_completion_flow(notebook_path, **base_paramete
 )
 async def jobs_runs_submit_by_id_and_wait_for_completion(
     databricks_credentials: DatabricksCredentials,
-    job_id: Optional[int] = None,
+    job_id: int = None,
```

Comment:
Is there a reason why the typing was changed?

Reply:
Yes, when submitting the flow above it makes a call to the '/2.1/jobs/run-now' endpoint which requires a job id

Reply:
If so, should the default = None be dropped?

Reply:
Yep, we don't need a default value for `job_id`
```suggestion
    job_id: int,
```